index.html

```diff
@@ -61,5 +61,194 @@ <h1 class="hero-intro">I’m Akorede. <br> Glad to see <br class="see-break"> yo
             </ul>
         </div>
     </header>
+    <main>
```

Comment:
Please use the same background color for you main section this is the color used in figma design #F7F7F9